gapis/trace/android/validate/validate.go

```diff
@@ -213,10 +213,10 @@ func ValidateGpuSlices(ctx context.Context, processor *perfetto.Processor) error
 		submissionIds := columns[2].GetLongValues()
 		for i := uint64(0); i < numRecords; i++ {
 			if commandBuffers[i] == 0 {
-				return log.Errf(ctx, nil, "Gpu slice %v has null command buffer", names[i])
+				return log.Errf(ctx, nil, "Gpu activity slice %v has null command buffer", names[i])
 			}
 			if submissionIds[i] == 0 {
-				return log.Errf(ctx, nil, "Gpu slice %v has null submission id", names[i])
+				return log.Errf(ctx, nil, "Gpu activity slice %v has null submission id", names[i])
```

Comment:
Since we're already touching this line...

```suggestion
				return log.Errf(ctx, nil, "GPU activity slice %v has null submission id", names[i])
```